### PR TITLE
Review html page names in core docs 2.1.6 - Closes #672

### LINKED ISSUE #672: Review html page names in Core docs 2.1.6
Review, if the file names of the docs pages can be improved.
example: API specification has api.html, configuration has config.html, Create custom transaction types has customize.html, reference to refererences.

 Review file names in 2.1.6 Core docs

Closes parent issue #532

### PR DESCRIPTION
**Pull request by chris529**
Closes #672 and parent epic issue #532

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
-* xref:interact-with-network.adoc[Interact with the network]
+* xref:interact-with-the-api.adoc[Interact with the network]
 * Setup
 ** xref:setup/application.adoc[Application]
-** xref:setup/commander.adoc[Commander application]
-** xref:setup/docker.adoc[Docker image]
-** xref:setup/source.adoc[Source code]
+** xref:setup/commander-application.adoc[Commander application]
+** xref:setup/docker-image.adoc[Docker image]
+** xref:setup/source-code.adoc[Source code]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: These should not be changed imho.
It is also inconsistent with the page names in the `Update` and `Management` section.
Please fix this also for the Core 3.0.0 docs

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/nav.adoc]: Hey Mona! OK, it should be done now, can you please check it. 👍

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please fix the existing conflicts

**Comment by chris529** [comment]
Ah yes.. OK conflicts should now be resolved 👍

File: modules/ROOT/pages/management/docker.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_lisk_snapshots: http://snapshots.lisk.io
 :url_config: management/configuration.adoc
 :url_config_cl: reference/config.adoc#clo
-:url_setup_docker_install: setup/docker.adoc#install
+:url_setup_docker_install: setup/docker-image.adoc#install
```
Comment (Tschakki) [inline modules/ROOT/pages/management/docker.adoc]: This should not be changed

**Review by Tschakki — CHANGES_REQUESTED** [review]
One small issue left

File: modules/ROOT/pages/management/docker.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/management/docker.adoc]: Oh yes, already done now..well spotted again Mona! 👍

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**